tests/managed/overloads.cs

```diff
@@ -277,4 +277,37 @@ public static AllOperatorsWithFriendly Negate (AllOperatorsWithFriendly c1)
 			return new AllOperatorsWithFriendly (-c1.Value);
 		}	
 	}
+
+	public class EqualOverrides {
+		public int Value { get; private set; }
+
+		public EqualOverrides (int val)
+		{
+			Value = val;
+		}
+
+		public static bool operator == (EqualOverrides a, EqualOverrides b)
+		{
+			return a.Value == b.Value;
+		}
+
+		public static bool operator != (EqualOverrides a, EqualOverrides b)
+		{
+			return a.Value == b.Value;
+		}
+
+		public override bool Equals (Object obj)
```

Comment:
this should be bound to `isEqual:` so it overrides the existing one from `NSObject`